framework/src/base/Moose.C

```diff
@@ -1193,3 +1193,5 @@ bool _deprecated_is_error = false;
 bool _throw_on_error = false;
 
 } // namespace Moose
+
+OStreamProxy & _console = Moose::out;
```

Comment:
This also seems like it should emit a bunch of shadowing warnings (two things named `_console` in the same scope) but we just don't have those warnings enabled.